OpenBullet/Pages/Dialogs/DialogSelectWordlist.xaml

```diff
@@ -147,7 +147,8 @@
             </ListView.View>
         </ListView>
 
-        <Button Grid.Row="1" x:Name="selectButton" Content="Select" Click="selectButton_Click" />
+        <Button Grid.Row="1" x:Name="selectButton" Content="Select" Click="selectButton_Click" Margin="0,0,26,0" />
+        <Image HorizontalAlignment="Left" Height="20" Margin="376,5,0,0" Grid.Row="1" VerticalAlignment="Top" Width="21" Source="/Images/Icons/folder.png" MouseDown="importWordlistButton_MouseDown"/>
```

Comment:
Did you drag and drop stuff again? Because I'm seeing custom margins on the Button and Image. Please only use XAML to build and use margin only when strictly necessary, otherwise use nested panels or grids to achieve what you want! The method is good nothing to say there ^_^